frontend/src/components/App.js

```diff
@@ -60,7 +60,7 @@ function App() {
   };
 
   const handleDocumentationClick = () => {
-    window.location = 'https://github.com/techno-tim/techno-broadlink';
+    window.location = 'https://automacaoraceland.pt';
```

Comment:
Can we revert this?

Reply:
oh, of course, I thought I had changed that. I'm so sorry

Reply:
NP!